### ISSUE TITLE
Objective DSSIM for time distributed sequence prediction

### ISSUE THREAD

**Issue by Mohamedemad4** [keras-team/keras-contrib#223]
Hello, I thought It would be cool to have a DSSIM loss that takes multiple Images per sample for sequence to sequence image prediction so here is the code for the loss function I am proposing, [github gist](https://gist.github.com/Mohamedemad4/f2a843db22a9b06f57cf38b1ee930cff)
What Is your feedback? and can this be added?
, thank you
